src/common/cockpitloopback.c

```diff
@@ -0,0 +1,156 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*- */
+
+/* GIO - GLib Input, Output and Streaming Library
```

Comment:
Did you copy this from GIO?
Otherwise we should use a more generic header here and also change the copyright date to 2015.